test/modules/TestResultCoordinator/Reports/CountingReport.cs

```diff
@@ -108,7 +115,20 @@ public bool IsPassedHelper()
                 },
                 () =>
                 {
-                    return this.TotalExpectCount == this.TotalMatchCount;
+                    // Product issue for C2D messages connected to edgehub over mqtt.
```

Comment:
All the `/ this.TotalExpectCount ` in this subroutine are prone to `DIVIDED BY 0` exception.

Reply:
We check that expect count is > 0 here:
https://github.com/Azure/iotedge/pull/5512/files#diff-64b0d03f03915eab5550575602fed2f602c3ad2f36feeb43d4b7929b8bb6027fR111